### DISCUSSION TITLE
稼動⇒稼働の一括変換

### DISCUSSION THREAD

**Discussion by pato-taityo** [pgsql-jp/jpug-doc#2693] [NGとする語]
### 問題の詳細

https://github.com/pgsql-jp/jpug-doc/pull/2664#discussion_r1257387046
このあたりでディスカッションさせていただいた件です。

同じ意味かつ意図無くぶれている場合は統一させてきたということで、稼動⇒稼働の一括変換を実施します。

翻訳作業が進行中ですので、なるべく影響の少ないタイミングを狙って実行する予定です。
それまでは翻訳対象外のファイルを個別に修正していきます。

### 貢献者として記載可否

記載（貢献者欄に書いてください）

### 貢献者名

伊東貢一

**Comment by noborus** [comment]
統一するということで、「稼動」はうっかりすると書いてしまうのでNGリストにも入れるのが良いでしょう。　#2436

**Comment by tatsuo-ishii** [comment]
稼働に統一で賛成ですが、この件に限らず、何故そうしたかの議論の経緯がコミットから終えるようになっていると嬉しいです。
（すでにそうなっているのであれば無視してください）
「稼動」が誤記で確定ならば話は単純ですが、必ずしも誤記ではないとの記述も存在するようなので。

**Comment by pato-taityo** [comment]
@noborus 
> 統一するということで、「稼動」はうっかりすると書いてしまうのでNGリストにも入れるのが良いでしょう。　#2436

ありがとうございます。jpug-doc 内の ".ng.list" でよかったでしょうか？
NGとした経緯は #2436 に記載するようでしたので、上記確認後に対応したいと思います。

**Comment by pato-taityo** [comment]
@tatsuo-ishii 
> 稼働に統一で賛成ですが、この件に限らず、何故そうしたかの議論の経緯がコミットから終えるようになっていると嬉しいです。

コメントをありがとうございました。以下に議論の経緯をリンクしたいと思います。
・PullRequestのコメント
・NGリスト追加時

> 「稼動」が誤記で確定ならば話は単純ですが、必ずしも誤記ではないとの記述も存在するようなので。

はい。おっしゃる通りです。調べる限り、誤記とは言い切れませんでした。
参考：
https://github.com/pgsql-jp/jpug-doc/pull/2664#discussion_r1257387046

これを踏まえたうえで、過去は同じ意味であればブレをなくして統一してきたと伺い、今回統一することになった次第です。
その過程は先の参考リンクにありますので、こちらを記載します。

**Comment by noborus** [comment]
> @noborus
> 
> > 統一するということで、「稼動」はうっかりすると書いてしまうのでNGリストにも入れるのが良いでしょう。　#2436
> 
> ありがとうございます。jpug-doc 内の ".ng.list" でよかったでしょうか？ NGとした経緯は #2436 に記載するようでしたので、上記確認後に対応したいと思います。

はい。トップの.ng.listに追加してください。
wikiに追加しました。 https://github.com/pgsql-jp/jpug-doc/wiki/%E4%BD%BF%E7%94%A8NG%E3%81%AA%E8%AA%9E

マージされたら、このissueをDiscussionsに変換するので、新たに書かなくても大丈夫です。

**Comment by pato-taityo** [comment]
@noborus

> マージされたら、このissueをDiscussionsに変換するので、新たに書かなくても大丈夫です。

こちらの対応をお願いしてもよろしいでしょうか…？
もし、私でもできるようであれば実施させていただきます。

**Reply by noborus** [reply]
discussions に変換しました。
